blue-note/patterns/black-image-with-text.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Title: Black Image with Text
+ * Slug: blue-note/black-image-text
+ * Categories: text
+ * Block Types: core/quote
+ */
+?>
+<!-- wp:columns {"style":{"spacing":{"blockGap":{"top":"0px","left":"0px"},"padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"}}},"backgroundColor":"white"} -->
+<div class="wp-block-columns has-white-background-color has-background" style="padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px"><!-- wp:column {"verticalAlignment":"bottom"} -->
+<div class="wp-block-column is-vertically-aligned-bottom"><!-- wp:cover {"url":"<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/Ethel-Waters.jpg","id":1888,"dimRatio":50,"minHeightUnit":"vh","contentPosition":"center center","style":{"spacing":{"padding":{"top":"var:preset|spacing|80","bottom":"var:preset|spacing|80","right":"1em","left":"1em"}}},"layout":{"type":"constrained","contentSize":"100%"}} -->
+<div class="wp-block-cover" style="padding-top:var(--wp--preset--spacing--80);padding-right:1em;padding-bottom:var(--wp--preset--spacing--80);padding-left:1em"><span aria-hidden="true" class="wp-block-cover__background has-background-dim"></span><img class="wp-block-cover__image-background wp-image-1888" alt="" src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/Ethel-Waters.jpg" data-object-fit="cover"/><div class="wp-block-cover__inner-container"><!-- wp:group {"style":{"dimensions":{"minHeight":""}},"layout":{"type":"constrained"}} -->
```

Comment:
It will look more like the design if we turn the opacity of the cover block down to 0